doc/conf.py

```diff
@@ -29,6 +29,9 @@
     'sphinx_copybutton',
```

Comment:
This is missing `sphinx_github_role` in the list of extensions :)

Reply:
gosh :man_facepalming:  ... thanks a lot @astrojuanlu